src/common/helpers/proposalTitleAndDescription/approvedIdentifiersTable.json

```diff
@@ -0,0 +1,1222 @@
+[
```

Comment:
This is awesome and is. a great temp solution until we've improved the design of how we do voting issues.  however, from playing with the UI iI realized that the list you tok from is. a bit out of date. in particular, you are missing `ACROSS-V2`

<img width="417" alt="image" src="https://user-images.githubusercontent.com/12886084/174752454-98a709d2-9156-4a3d-8d8f-e659a6a9b8da.png">

see the umip here: https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-157.md

Reply:
actually the right solution is to update the docs then re-run the script, right?

Reply:
Yeah that's right, I'm just working with what's defined in the docs since we need to keep those in sync. I've set the script up to download that file with one command, so for now we can just run that script whenever the docs are updated. Will be easy to automate this in the future as well.